odoo/__init__.py

```diff
@@ -80,6 +80,21 @@ def registry(database_name=None):
         database_name = threading.currentThread().dbname
     return modules.registry.Registry(database_name)
 
+# ----------------------------------------------------------
+# module babel hack
+# make sure ICU [world] territory is "001"
+# whereas "unitag" notation is "AA"
+# http://www.localeplanet.com/icu/ar-001/index.html
+# ----------------------------------------------------------
+import babel
+_babelCoreParseLocale = babel.core.parse_locale
+def _babelCoreParseLocale_unitag(identifier, sep='_'):
+    lang, territory, script, variant = _babelCoreParseLocale(identifier, sep)
+    territory = '001' if territory == 'AA' else territory
+    return lang, territory, script, variant
+
+babel.core.parse_locale = _babelCoreParseLocale_unitag
```

Comment:
Could we move the whole "babel hack" block under the "lib UTC hack" one, for consistency?